releasenotes-dca/notes/add-k8s-gradual-rollout-support-315926f2d3c1b6d3.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Introducing support for gradual rollout for K8s SSI deployments.
+    This will allow the injector and tracer library releases to be rolled out by subsets of customers,
+    and prevent the rollout to more customers if there are issues detected.
```

Optional suggestion!

```suggestion
    Added support for gradual rollout in K8s SSI deployments.
This enables injector and tracer library releases to subsets of customers, with the ability to pause if issues are detected.
```